.dotnet/src/Custom/FineTuning/GeneratorStubs.cs

```diff
@@ -8,29 +8,69 @@ internal partial class CreateFineTuningJobRequest { }
 [CodeGenModel("CreateFineTuningJobRequestHyperparameters")]
 internal partial class CreateFineTuningJobRequestHyperparameters { }
 
```

Comment:
Curious: what's the reason for removing the "readonly" keyword from these? 🤔

Reply:
No idea; my best guess is a bad string replacement because I have no recollection of changing those! Replaced all instances of `internal partial struct` with `internal readonly partial struct`.